css-color/Overview.bs

```diff
@@ -1714,7 +1717,7 @@ Profiled, Device-dependent Colors</h2>
 	or an RGB colorspace (such as ProPhoto <!-- ref -->, <!-- would be good to mention AdobeRGB 1998 here too, need to check trademark and copyright issues first --> widely used by photographers), or any other color or monochrome output device which has been characterized.
 	In addition, for convenience,
 	CSS provides two predefined RGB color spaces:
-	DCI P3 [[!DCI-P3]],
+	p3d65 [[!DCI-P3]],
```

Comment:
I would suggest displayp3 or display-p3. Here's the ICC file that ships with macOS Sierra.
[Display P3.icc.zip](https://github.com/w3c/csswg-drafts/files/506728/Display.P3.icc.zip)